.setup/bin/setup_sample_courses.py

```diff
@@ -508,8 +505,6 @@ def create_gradeable_submission(src, dst):
     a .zip), then we will extract the contents of the source (using ZipFile) to the destination, else we
     just do a simple copy operation of the source file to the destination location.
 
-    At this point, if we created a zip file (as part of that first step), we remove it from the /tmp directory.
```

Comment:
Please undo this change as this comment is still true even after your edits to the code.

Reply:
Did the change in 94841d9